src/lib/moderation.ts

```diff
@@ -0,0 +1,28 @@
+export interface warnLog {
```

Comment:
Types should all use title case.
```suggestion
export interface WarnLog {
```